docs/design/datacontracts/RuntimeTypeSystem.md

```diff
@@ -4,46 +4,80 @@ This contract is for exploring the properties of the runtime types of values on
 
 ## APIs of contract
 
-A `MethodTable` is the runtime representation of the type information about a value.  Given a `TargetPointer` address, the `RuntimeTypeSystem` contract provides a `MethodTableHandle` for querying the `MethodTable`.
+A `TypeHandle` is the runtime representation of the type information about a value which is represented as a TypeHandle.
+Given a `TargetPointer` address, the `RuntimeTypeSystem` contract provides a `TypeHandle` for querying the details of the `TypeHandle`.
+
 
 ``` csharp
-struct MethodTableHandle
+struct TypeHandle
 {
     // no public properties or constructors
 
-    internal TargetPointer Address { get; }
+    public TargetPointer Address { get; }
+    public bool IsNull => Address != 0;
+}
+
+struct TypeHandle
+{
+}
+
+internal enum CorElementType
+{
+    // Values defined in ECMA-335 - II.23.1.16 Element types used in signatures
+    // + 
+    Internal = 0x21, // Indicates that the next pointer sized number of bytes is the address of a TypeHandle. Signatures that contain the Internal CorElementType cannot exist in metadata separate from a live process.
```

Comment:
Oh, sorry.. I meant to express that they can't be in the disk format or pulled anywhere from a saved metadata block. (Also this detail won't be used until we implement `GetMethodDescName`, but since I was adding CorElementType, I added it at this time.)